services/brig/src/Brig/Team/DB.hs

```diff
@@ -57,34 +58,35 @@ data InvitationInfo = InvitationInfo
 
 insertInvitation :: MonadClient m
                  => TeamId
+                 -> Maybe Role
```

Comment:
Btw, just a small note here: we could certainly change the `insertInvitation` to accept `Role` and the inviter: `UserId`, since those will always exist after this deployment;

We still need those `Maybe`s on the read path but wondering if we can already ensure they get written during the creation of an invitation (you could also change the `FromJSON` instance of `InvitationRequest` to default to member, we do that in some places). Given that invitations only live on our DB for 2 weeks in production, the read path could even be changed after that.

Not necessary to change it, just food for thought, not sure what the general opinion about this topic.

Reply:
`Role`: agreed (even though it doesn't seem important to me).  `UserId`: what about SCIM?

Reply:
Yes, please let's not have `UserId`. Ideally we should have `UserId | ScimTokenId` for auditabilibilibility purposes, though then we might also need to think in the future about deactivating SCIM tokens instead of deleting them.

Reply:
While I'm at it, we might also want to store info about who created which SCIM token. Auditabilibilibility!

Reply:
My bad, forget about the `UserId`